### PR TITLE
Allow parameterless commands to build correctly

### LINKED ISSUE #253: Parameterless Command in Group/Module injects extra space
```csharp
[Module("prune"}]
public class PruneCommands {
    [Command]
    public async Task Prune(IUserMessage message) {
    }
}
```

Produces a Command with Text `prune `. This is required in addition to the normal space separator and registers an unknown command error if the extra space is not present.

The same happens with GroupAttribute as well.

### PR DESCRIPTION
**Pull request by FiniteReality**
Resolves #253 and works correctly with aliases/etc.

### CODE REVIEW

File: src/Discord.Net.Commands/Module.cs [hunk 1]
```diff
@@ -63,9 +61,15 @@ private void SearchClass(TypeInfo parentType, object instance, List<Command> com
                 {
                     string nextGroupPrefix;
                     if (groupAttrib.Prefix != null)
+                    {
+                        if (groupPrefix != "")
```
Comment (RogueException) [inline src/Discord.Net.Commands/Module.cs]: This is inside the type loop. Wont this add multiple spaces if there are multiple commands in a class?

Comment (FiniteReality) [inline src/Discord.Net.Commands/Module.cs]: Oh boy, it probably will. Let me fix that.

File: src/Discord.Net.Commands/Module.cs [hunk 2]
```diff
@@ -63,9 +61,16 @@ private void SearchClass(TypeInfo parentType, object instance, List<Command> com
                 {
                     string nextGroupPrefix;
                     if (groupAttrib.Prefix != null)
```
Comment (FiniteReality) [inline src/Discord.Net.Commands/Module.cs]: Wait, this still introduces a bug. Whoops.

**Comment by FiniteReality** [comment]
I think there's still a bug in Module.cs, or at least a logic error. I'll look into it.

**Comment by FiniteReality** [comment]
That should fix that error; everything should be fine now.

**Pull request merged**